fairer-iOS/fairer-iOS/Global/Util/RootHandler.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  RootHandler.swift
+//  fairer-iOS
+//
+//  Created by 김규철 on 2023/04/18.
+//
+
+import UIKit
+
+final class RootHandler {
+    static let shared = RootHandler()
+    
+    func change() {
+        let LoginViewController = OnboardingNameViewController()
```

Comment:
p2;
```suggestion
        let loginViewController = OnboardingNameViewController()
```
사소한 거지만,,, 소문자로 시작하는 건 어떤가요?